class-events-manager-securesubmit.php

```diff
@@ -137,7 +138,7 @@ public function __construct() {
 
             add_filter( 'em_wp_localize_script', array( $this, 'em_wp_localize_script' ), 10, 1 );
             add_filter( 'em_booking_form_action_url', array( $this, 'force_ssl' ), 10, 1 );
-            wp_enqueue_script('hps_em_securesubmit_library', plugins_url('jquery.securesubmit.js',__FILE__), array('jquery'));
+            wp_enqueue_script('securesubmit', plugins_url('jquery.securesubmit.js',__FILE__), array('jquery'));
```

Comment:
@SenthilKumar-86 I'd like to keep this as it was. The change to `hps_em_securesubmit_library` was originally made to combat conflicts with our other plugins that have already upgraded to our pure JavaScript `securesubmit.js`